test/testvulkan.c

```diff
@@ -1176,7 +1184,7 @@ int main(int argc, char *argv[])
     {
         SDL_Log("%2.2f frames per second\n", ((double)frames * 1000) / (now - then));
     }
-    quit(0);
+    shutdownVulkan(SDL_FALSE);
```

Comment:
This misses `SDLTest_CommonQuit(state);` ?

Reply:
Thanks, updated.